src/detailed/authentication_and_encryption.md

```diff
@@ -0,0 +1,48 @@
+# Authentication & Encryption
+This page details how Authentication & Encryption may work for Eludris.
+
+## The Token
+
+Eludris uses JWT tokens to authenticate users. These tokens are required for nearly every interaction. Connection to the Gateway or trying to interact with the API? You'll need a token!
+The JWT tokens used within our services use a cryptographically secure pseudo-random string with HC128 as a secret. 
+If you wish to get a new token send an HTTP request to `/auth` with your email and password.
+
+## End-To-End-Encryption
+
+End-To-End-Encryption (we'll be calling it E2EE for now on) will be available to private communities, private GDMs (group direct messages) and direct messages (DMs) between friends.
+
+### E2EE Implementation
+
+To introduce this, every user has a personal, unique, public and private key provided.
```

Comment:
A tad nitpicky, but I feel like this reads as if there were four keys, maybe something like `[...] every user is provided a personal and unique pair of a public and private key.`?